### ISSUE TITLE
Demo One (Version 6) PAL not loading

### ISSUE THREAD

**Issue by Desz5** [sahlberg/pop-fe#41]
The demo i'm trying to convert is Demo One (Version 6) - PBPX-95007
When creating a PKG from the bin/cue results in a blank screen at startup.
I have this demo running in Multiman on 576p and the emulator set to ps1_netemu.
Forcing PAL also results in a black screen.
I believe i can set Force NTSC to use the ps1_netemu but this results in NTSC speeds.
Or do i have to create a ps3config for this specific game?
Is there a way we can choose which emulator (ps1_emu or ps1_netemu) we want to use in pop-fe?
Thanks!

**Comment by sahlberg** [comment]
Can you try ForceNTSC on the very latest build in the actions tab?
I originally only added the "slow down" config in pop-fe but not pop-fe-ps3 so it should run at correct speed now if you use force ntsc from pop-fe-ps3.

Otherwise we may need a ps3config for this game, unless it is a bug in pop-fe breaking something when it generates the pkg.

I think we will soon have a way to select a different emulator. I think mrjaredbeta mentioned a config that would select ps1_netemu instead. Once he gives me that config I will add a checkbob in pop-fe-ps3 to select it.

You possibly also want to ask on https://www.psx-place.com/threads/ps1_netemu-external-configs-and-ad-hoc-params.41504/ because I don't know much about the configs and only add what mrjaredbeta tells me to add :-)


I did create a small tool to re-build ISO.BIN.DAT/EDAT with a new config to speed up testing. So if you want to test different configs I can give you the link to that tool.

**Comment by Desz5** [comment]
I tried using the ForceNTSC option but it still results in a black screen at startup.
Does Pop-fe use ps1_emu for all PKG's but ps1_netemu only when using ForceNTSC?
The ps3configs are only used by ps1_netemu right?
So when forcing NTSC the ps1_netemu is used but run under 480p and not 576p.
I've tried to ForceNTSC and searching for the file:

[pop-fe/pop-fe.py](https://github.com/sahlberg/pop-fe/blob/275827993e9fed21d97355d65d8586053de84162/pop-fe.py#L576)

Line 576 in [2758279](https://github.com/sahlberg/pop-fe/commit/275827993e9fed21d97355d65d8586053de84162)
'data': 1},

So i could change the 1 to 2 using 576p, but i couldn't find the file.
It would be great if we could choose the resulution, 480p or 576p and choose the emulator, ps1_emu or ps1_netemu.
And it's great that mrjaredbeta is working on that.
I will reply to the thread on psx-place, thank you!
